source/blog/2019-10-23-summer-fall.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: post
+title: Fall Update 🍂
+published: true
+post_author:
+  display_name: Yufan Gong
+  twitter: yufangong
+tags: Finagle, Finatra, Util, Scrooge, TwitterServer
+---
+
+Hi Finaglers,
+
+Fall has arrived in the Northern Hemisphere 🌍🌎🌏, and in San Francisco, we are enjoying the light breeze🎐and warm sunshine☀️. Here are some quick highlights of our work from July, August, and September. If you want to keep an 👁 on our most recent updates, please watch our changelogs ([Util](https://github.com/twitter/util/blob/develop/CHANGELOG.rst), [Scrooge](https://github.com/twitter/scrooge/blob/develop/CHANGELOG.rst), [Finagle](https://github.com/twitter/finagle/blob/develop/CHANGELOG.rst), [Twitter-Server](https://github.com/twitter/twitter-server/blob/develop/CHANGELOG.rst), [Finatra](https://github.com/twitter/finatra/blob/develop/CHANGELOG.rst)).
```

Comment:
Is the `*` after sunshine an emoji?

Reply:
yeah that should be sun after render?? I guess